R/text_line_dataset.R

```diff
@@ -169,7 +169,8 @@ make_csv_dataset <- function(file_pattern, batch_size,
   # valdiate version
   validate_tf_version("1.8", "make_csv_dataset")
 
-  tf$contrib$data$make_csv_dataset(
+  ## Gather arguments common to all tensorflow versions
+  opts = list(
```

Comment:
Our normal convention is to name argument lists as `args` rather than `opts`, could you make this change?

Reply:
Please also use `<-` for assignment throughout.